crafter-search-opensearch/src/main/java/org/craftercms/search/opensearch/impl/OpenSearchAdminServiceImpl.java

```diff
@@ -229,7 +229,7 @@ public void deleteIndexes(final String aliasName) throws OpenSearchException {
     protected void doDeleteIndexes(RestHighLevelClient client, String aliasName) {
         try {
             GetAliasesResponse indices = client.indices().getAlias(
-                    new GetAliasesRequest(aliasName + "*"),
+                    new GetAliasesRequest(aliasName),
```

Comment:
Let's discuss this one.

Reply:
For serverless, the alias will be the site name, no suffix. So `siteName*` will match the other indices for that site: `siteName-authoring` and `siteName-preview`.
Moreover, it will match anything prefixed with `siteName`. e.g.: `siteName-copy-3-authoring`, `siteName-duplicate-preview`
This appeared in my testing when the delete index hook was added to s3 target template

Reply:
This is a good catch.